tests/unit/test_repo_manager.py

```diff
@@ -182,3 +184,18 @@ def testRemoveVariantSet(self):
         with self.assertRaises(exceptions.RepoManagerException):
             self.repoManager.removeVariantSet(
                 'dataset1', paths.variantSetName)
+
+    def testAddOntologyMap(self):
+        with self.assertRaises(exceptions.RepoManagerException):
+            self.repoManager.addOntologyMap(paths.bamPath, 'link')
+        self.repoManager.addOntologyMap(
+            paths.ontologyPath, 'link')
+
+    def testRemoveOntologyMap(self):
+        with self.assertRaises(exceptions.RepoManagerException):
+            self.repoManager.removeOntologyMap(
+                paths.ontologyName)
+        self.repoManager.addOntologyMap(
+            paths.ontologyPath, 'link')
+        self.repoManager.removeOntologyMap(
```

Comment:
check that the repo manager throws an error after an additional removeOntologyMap call (i.e. when there should be no extant ontologyMap to remove)